symfony/composer.json

```diff
@@ -12,14 +12,13 @@
     "algolia/search-bundle": "^4.0",
     "symfony/console": "^5.0",
     "symfony/dotenv": "^5.0",
-    "symfony/flex": "^1.1",
+    "symfony/flex": "^1.8.4",
     "symfony/orm-pack": "^1.0",
     "symfony/yaml": "^5.0"
   },
   "require-dev": {
     "doctrine/doctrine-fixtures-bundle": "^3.1",
-    "phpunit/phpunit": "^8.4",
-    "roave/security-advisories": "dev-master",
+    "phpunit/phpunit": "^9.1",
```

Comment:
I think it's nice to keep the dependency "roave/security-advisories'. Moreover, I don't like the versioning of composer.lock

Reply:
> I don't like the versioning of composer.lock

This is a real application that people expect to pull and work with the playground. If we don't commit the composer.lock, we take the risk of shipping a playground that don't work once people perform `composer install`.